wasmtime-jni/src/wasi_config.rs

```diff
@@ -35,5 +36,16 @@ pub fn ctx_from_java(env: &JNIEnv, obj: JObject) -> Result<WasiCtx> {
         let dir = preopen_dir(&host_path)?;
         builder.preopened_dir(dir, &guest_path);
     }
+
+    if let Some(file) = utils::get_optional_string_field(env, obj, "stdoutFile")? {
+        let file = File::create(file)?;
```

Comment:
Create a file at the specified path as wasmtime-c-api does.
 https://github.com/bytecodealliance/wasmtime/blob/40d546c420c1e6c4909fc80e048dd1854fda1cd6/crates/c-api/src/wasi.rs#L139-L142